### PR TITLE
Enable IPv6 NAT

### PR DESCRIPTION
**Pull request by wrridgwa**
This pull request addresses this open issue https://github.com/moby/moby/issues/25407 
Previously when IPv6 was enabled on Docker, ip6tables weren't manipulated to do Port Forwarding and IP masquerading. This PR addresses that in a way similar to the IPv4 implementation. 

Relies on the following Moby PR to bring the EnableIPv6 flag through to libnetwork: https://github.com/moby/moby/pull/35252

When combined with the Docker pull request, the fix can be verified after starting a container in bridge mode with port forwarding and IPv6 enabled:
1. Compare the `iptables -t nat -L` command with the `ip6tables -t nat -L` command 
2. Compare `iptables -t filter -L` command with `ip6tables -t filter -L` command
3. Ping6 from within a running container to an outside device.

### CODE REVIEW

File: drivers/bridge/port_mapping.go [hunk 1]
```diff
@@ -5,13 +5,15 @@ import (
...
+	"strings"
 
 	"github.com/docker/libnetwork/types"
 	"github.com/sirupsen/logrus"
 )
 
 var (
-	defaultBindingIP = net.IPv4(0, 0, 0, 0)
+	defaultBindingIP   = net.IPv4(0, 0, 0, 0)
+	defaultBindingIPV6 = net.ParseIP("::1")
```
Comment (chrisnew) [inline drivers/bridge/port_mapping.go]: Are you sure about `::1`? This is localhost in IPv6 where 0.0.0.0 means any address in IPv4.

Comment (wrridgwa) [reply drivers/bridge/port_mapping.go]: Ah, you are right. This should be `::`.

Comment (wrridgwa) [reply drivers/bridge/port_mapping.go]: Fixed.

Comment (chrisnew) [reply drivers/bridge/port_mapping.go]: :man_dancing: awesome! :+1:

**Comment by wrridgwa** [comment]
What can I do to move this pull request forward?

**Comment by fcrisciani** [comment]
@wrridgwa I would start with rebasing on top of latest master and squash all your changes into a single commit, right now the PR has a lot of extra stuff

**Comment by wrridgwa** [comment]
@fcrisciani Should I just create a new pull request and put all of my stuff into one commit? Because of the commit signing issue that I ran into initially and the fact that my commits aren't all next to each other, its going to be hard to clean this up.

**Comment by wrridgwa** [comment]
@fcrisciani I created a new pull request which is much cleaner: https://github.com/docker/libnetwork/pull/2023

**Comment by antoineco** [comment]
@wrridgwa could you please close this for clarity?

**Pull request closed without merging**